### DISCUSSION TITLE
How to Add/Remove Favorite emotes & Put Twitch Global emotes tab at top

### DISCUSSION THREAD

**Discussion by cowls192** [night/betterttv#6712] [Ideas]
Its so annoying i have to be the only one to come up with this issue its not that hard to fix it

**Comment by dclstn** [comment]
These features already exist within our emote-menu. ALT + Left-Click to favourite an emote, and the categories in the sidebar can be dragged and dropped to reorder how you plrease.

**Reply by cowls192** [reply]
Thank you so much for response
